cmd/kube-proxy/app/server.go

```diff
@@ -477,6 +477,11 @@ func NewProxyServer(config *componentconfig.KubeProxyConfiguration, cleanupAndEx
 			return nil, fmt.Errorf("unable to read IPTables MasqueradeBit from config")
 		}
 
+		nodeIP, err := getNodeIP(client, hostname)
+		if err != nil {
+			return nil, fmt.Errorf("unable to fetch node IP: %v", err)
```

Comment:
This particular node address is used in one place in constructing iptables rules, if my tracing of the code is correct.